test/functionalTest/cases/1367_time_measures/time_measure_stats.test

```diff
@@ -31,7 +31,7 @@ brokerStart CB 101,38 IPv4 -reqTimeStat
 --SHELL--
 
 #
-# 01. Initial call to GET /statistics (gives NO time stats as LAST /statistics is excluded)
+# 01. Pre-call to GET /statistics to give VALGRIND a possibility
```

Comment:
ok, I'll put this explanation:

The valgrind test suite uses a call to "GET /version" to check that the broker is alive. This request is reflected in the statistics so the first call to "GET /statistics" gets the "timing" stuff (as there are non-zero counters) when under valgrind. When running as a normal functest, there is no prevoius "GET /version" and thus no timing counters.
Now, we have no means to do REGEXPs about missing lines so with this call to "GET /statistics", a first request is forced and the second call will give "timing" counters, both as normal functest and when under valgrind.